README.md

```diff
@@ -9,9 +9,9 @@
 [![Translation status](https://weblate.iobroker.net/widgets/adapters/-/ical/svg-badge.svg)](https://weblate.iobroker.net/engage/adapters/?utm_source=widget)
 [![Downloads](https://img.shields.io/npm/dm/iobroker.ical.svg)](https://www.npmjs.com/package/iobroker.ical)
 
-ioBroker.ical is an adapter for the ioBroker automation platform focusing on iCalendar files, widely used for storing and sharing calendar data. It allows users to read iCalendar files locally or from a specified URL and parse it.  
+ioBroker.ical, based on iCal Adapter for [CCU.IO](https://github.com/hobbyquaker/ccu.io/tree/master/adapter/ical) from vader722, is an adapter for the ioBroker automation platform focusing on iCalendar files, widely used for storing and sharing calendar data. It allows users to read and parse iCalendar files locally or from a specified URL.  
```

Comment:
This "based on" is years ago, I think it is no longer relevant enough to be in the very first sentence beause this will not bring any benefit for a reader these days.